indico/modules/events/persons/client/js/AuthorsList.jsx

```diff
@@ -145,7 +144,7 @@ export function AuthorsList({eventId, sourceIds, objectContext, onClose}) {
       <Modal.Actions>
         <EmailParticipantRolesButton
           eventId={eventId}
-          persons={selectedPersons.filter(id => isVisible(eventPersons.get(id)))}
+          persons={selectedPersons.filter(id => isVisible(data.eventPersons[id]))}
```

Comment:
Should we keep using a Map of eventPersons here for performance? Or is an object fine?

Reply:
I think we just used it because we needed something mapping-like. Even with a few thousand EventPersons I doubt there will be a noticeable difference (object key lookup should be around O(1) as well)